balboa-http/src/it/scala/com/socrata/balboa/server/MetricsIntegrationTest.scala

```diff
@@ -2,34 +2,139 @@ package com.socrata.balboa.server
 
 import java.net.URL
 
+import com.socrata.balboa.metrics.Metric.RecordType
+import com.socrata.balboa.metrics.{Metric, Metrics}
+import com.socrata.balboa.metrics.data.{DataStoreFactory}
 import com.stackmob.newman.ApacheHttpClient
 import com.stackmob.newman.dsl.GET
+import com.stackmob.newman.response.HttpResponse
 import com.stackmob.newman.response.HttpResponseCode.{BadRequest, NotFound, Ok}
-import org.json4s.jackson.JsonMethods.parse
-import org.scalatest.{FlatSpec, Matchers}
+import org.json4s.jackson.JsonMethods.{parse, pretty, render}
+import org.scalatest.{BeforeAndAfterEach, FlatSpec, Matchers}
+import scala.language.implicitConversions
 
-import scala.concurrent._
+import scala.concurrent.Await
 import scala.language.postfixOps
 
-class MetricsIntegrationTest extends FlatSpec with Matchers {
+class MetricsIntegrationTest extends FlatSpec with Matchers with BeforeAndAfterEach {
   implicit val httpClient = new ApacheHttpClient
+  val ds = DataStoreFactory.get()
+  val testMetName = "testMetricsPersisted"
+  val testEntPrefix = "testMetricsEntity"
+  var testEntName = ""
```

Comment:
Prefer full names over abbreviations, except in the case of really well known abbreviations (db = database kind of well known.)